test/OpenSpaceToolkit/Astrodynamics/Trajectory/Validation/Propagator.cross.validation.cpp

```diff
@@ -768,7 +770,29 @@ TEST_P(OpenSpaceToolkit_Astrodynamics_Validation_CrossValidation_Thruster, Force
         propagatorWithTabulated.calculateStatesAt(initialState, instantArray);
 
     // Create maneuver from tabulated and propagator with maneuver
-    const Maneuver maneuver = Maneuver::TabulatedDynamics(tabulated);
+    Array<State> maneuverStateArray = Array<State>::Empty();
```

Comment:
Thinking about this, we actually could have kept this `TabulatedDynamics` static method and just required that states with position and velocity coordinate subsets be provided along with it

Reply:
That is a strange tabulated dynamic where you're tabulating states. We could add it but I don't see any value, and it wasn't being used anywhere so not sure it's worth.

Reply:
Yeah fair I guess that is a little awkward. In the future we may want to revisit this with our plans to include acceleration subset in all states. Because all accelerations are applied at a certain state